subprojects/docs/src/docs/userguide/authoring-builds/multi_project_builds.adoc

```diff
@@ -15,40 +15,144 @@
+└── settings.gradle.kts
+----
+=====
+====
 
-Let's start with a very simple multi-project build. Gradle is a general purpose build tool at its core, so the projects don't have to be Java projects. Our first examples are about marine life.
+This is also the recommended project structure for starting any Gradle project.
+The <<build_init_plugin#,build init plugin>> also generates skeleton projects that follow this structure - a root project with a single subproject.
 
+Note that the root project does not have a Gradle build file, only a settings file that defines the subprojects to include.
+====
+include::sample[dir="snippets/multiproject/basic-multiproject/groovy",files="settings.gradle[]"]
+include::sample[dir="snippets/multiproject/basic-multiproject/kotlin",files="settings.gradle.kts[]"]
+====
 
-[[sec:configuration_and_execution]]
-=== Configuration and execution
+In this case, Gradle when invoked will look for a build file in the `app` directory.
 
-<<build_lifecycle.adoc#sec:build_phases,Build phases>> describes the phases of every Gradle build. Let's zoom into the configuration and execution phases of a multi-project build. Configuration here means executing the `build.gradle` (or `build.gradle.kts`) file of a project, which implies e.g. downloading all plugins that were declared using '```apply plugin```' or a `plugins` block. By default, the configuration of all projects happens before any task is executed. This means that when a single task, from a single project is requested, _all_ projects of multi-project build are configured first. The reason every project needs to be configured is to support the flexibility of accessing and changing any part of the Gradle project model.
+We can view the structure of a multi-project build by running the `gradle projects` command.
+----
+> gradle -q projects
+include::{snippetsPath}/multiproject/basic-multiproject/tests/projects.out[]
+----
 
+Let's say the `app` subproject is a Java application by applying the <<application_plugin#,application plugin>> and configuring the main class:
+====
+include::sample[dir="snippets/multiproject/basic-multiproject/groovy",files="app/build.gradle[]"]
+include::sample[dir="snippets/multiproject/basic-multiproject/kotlin",files="app/build.gradle.kts[]"]
+.app/src/main/java/com/example/Hello.java
+----
+include::{snippetsPath}/multiproject/basic-multiproject/groovy/app/src/main/java/com/example/Hello.java[]
+----
+====
 
-[[sec:configuration_on_demand]]
-==== Configuration on demand
+Finally, we can run the application by executing the `run` task from the <<application_plugin#,application plugin>>.
+----
+> gradle -q run
+include::{snippetsPath}/multiproject/basic-multiproject/tests/run.out[]
+----
+
+And that's how simple it is to create a basic multi-project build.
+
+Next, we will look at multi-project builds that contain more than one subproject,
+how build logic can be shared across subprojects and how subprojects can depend on one another.
+
+
```

Comment:
I am tempted to expand here (in another section) about the structure of multi-project builds, tweaks that can be done in `settings.gradle` - how the subprojects can be put in a separate directory, how subproject hierarchies work, how to configure custom build file names for subprojects etc, but then realized this is probably something that the next recommendation about structuring subprojects should tackle.